docs/bigtable-data-api.rst

```diff
@@ -0,0 +1,328 @@
+Data API
+========
+
+After creating a :class:`Table <gcloud.bigtable.table.Table>` and some
+column families, you are ready to store and retrieve data.
+
+Cells vs. Columns vs. Column Families
++++++++++++++++++++++++++++++++++++++
+
+* As we saw before, a table can have many column families.
+* As we'll see below, a table also has many rows (specified by row keys).
```

Comment:
Avoid "we" ("you" is fine). This is better stated as "As described below, a table can also have many rows which are specified by row keys."